jobsupervisor/monit_job_supervisor.go

```
@@ -181,6 +181,15 @@ func (m monitJobSupervisor) Status() (status string) {
 	return
 }
 
+func (m monitJobSupervisor) ServiceStatus() (stats []boshmonit.Service) {
+	monitStatus, err := m.client.Status()
+	if err != nil {
+		return
```

do not swallow errors. should be returned.